modules/thunder_updater/updates.yml

```diff
@@ -20,3 +20,7 @@ v1.1:
         '#title': 'Remove language filter from media browsers'
         '#description': '<p>Language filtering should be removed in views used by media browsers. Media entities don''t have relation to language and filter doesn''t have effect.</p><p>In order to do that, you should make following steps:<ol><li>Go on views administration page.</li><li>Search for view that you want to edit (fe. "Image Browser", "Media Browser").</li><li>Click <b>Edit</b> to modify view.</li><li>On view editing page under <b>FILTER CRITERIA</b> find link <b>Media: Translation language (exposed)</b> and click it.</li><li>In pop-up dialog click <b>Remove</b> in bottom action bar.</li><li>Save view changes by clicking <b>Save</b> button.</li></ol> </p><p>Repeat described process for all views you need.</p>'
         views_page: { '#text': 'Views administration page', '#url': '/admin/structure/views' }
+    v1_1__image_video_paragraph:
+        '#title': 'Splitting media paragraph into video and image paragraph'
+        '#description': '<p>We think for editors it is more clearer to have a seperate video and image paragraph. So we installed that in an update hook for you. You can now decide by yourselve, if you want to use them.</p>'
```

Comment:
It should be made clear, that the  video and image paragraphs are meant as an alternative to the previously existing media paragraph.